include/Gaffer/ValuePlug.h

```diff
@@ -176,6 +176,11 @@ class GAFFER_API ValuePlug : public Plug
 		static void clearHashCache();
 		//@}
 
+		/// Control a very expensive mode which tests whether the hash cache
+		/// is cleared correctly.
+		static void setHashCacheValidation( bool hashCacheValidation );
+		static bool getHashCacheValidation();
```

Comment:
As discussed over chat, it would be good to reduce the expense of this so that we have the option of turning it on in interactive sessions.